usaspending_api/common/recipient_lookups.py

```diff
@@ -81,3 +82,51 @@ def recipient_is_child(recipient_record: dict) -> bool:
 
 def combine_recipient_hash_and_level(recipient_hash, recipient_level):
     return "{}-{}".format(recipient_hash, recipient_level.upper())
+
+
+def annotate_recipient_id(field_name, queryset):
```

Comment:
Should we exclude the SPECIAL_CASES here too?

Reply:
Holy cow.  Good catch.  Huge oversight.  Thanks.